docs/README.md

```diff
@@ -280,24 +280,34 @@ aws cloudfront  create-invalidation --profile docs.docker.com --distribution-id
 aws cloudfront  create-invalidation --profile docs.docker.com --distribution-id $DISTRIBUTION_ID --invalidation-batch '{"Paths":{"Quantity":1, "Items":["/v1.1/reference/api/docker_io_oauth_api/"]},"CallerReference":"6Mar2015sventest1"}'
 ```
 
-### Generate the man pages for Mac OSX
+### Generate the man pages 
```

Comment:
Why is this info here and in the man/README.md ?  Seems like the would become out of sync pretty easily.  Would it make more sense to just have a pointer from here to the one in the 'man' dir?